tests/std/tests/P0122R7_span/test.cpp

```diff
@@ -16,6 +16,16 @@
 
 using namespace std;
 
+static_assert(is_standard_layout_v<span<int>>);
+static_assert(is_standard_layout_v<span<int, 4>>);
+
+#if __has_cpp_attribute(no_unique_address)
+static_assert(sizeof(span<int>) == sizeof(void*));
+#else // ^^^ has no_unique_address ^^^ / vvv no no_unique_address vvv
+static_assert(sizeof(span<int>) == sizeof(void*) + sizeof(size_t));
+#endif // no no_unique_address vvv
+static_assert(sizeof(span<int, 4>) == sizeof(void*) + sizeof(size_t));

```

Comment:
I think those checks are backwards: `sizeof(span<int>)` should always be  `sizeof(void*) + sizeof(size_t)` and `sizeof(span<int, 4>)` would depend on the availability of `no_unique_address`.

Reply:
Argh
yepp you are right ^^ thanks MSVC